### DISCUSSION TITLE
八奈见杏菜_:(´□`」 ∠):_

### DISCUSSION THREAD

**Discussion by manggis-silk** [share121/inter-knot#1820] [其他]
来了😡我这一辈子就是被八奈见杏菜害了😭，没法跟正经妹子处事😭，跟妹子吃饭的时候，总是在想，她要是杏菜就好了🤤，有憨憨的表情那种，送她回家的时候，总是在想，她要是杏菜多好，能和她有说有笑的回去就好了，坐在她的家里的沙发上的时候，她要是靠过来时给你讲这个月的账单又要拿便当来还怎么办？然后妹子穿着四个蝴蝶结的校服出来了，问你喜不喜欢，你拿出了手机打开杏菜的图片说当然，我最喜欢杏菜了，朴素但又十分耐看有着青春女生特有的可爱和少女感加上怀有情愫半推半就的傲娇，这是何等直击内心的可爱！！虽然杏菜一直给人很可靠的感觉，但遇到自己不擅长的事，还是会不知所措，笨拙又慌张，一眼就能被看出想法的老八很可爱！和你在一起的时候，声音比起跟其他学生面前，音调更高和温柔的杏菜也很可爱！有些得意地“抱怨”着【你根本就离不开我！】的杏菜超级可爱😇！结婚😍！结婚😍！还是踏码的结婚[喜欢]！想到结婚后有杏菜在背后辅佐，补偿着生活的开销，心里就感到非常踏实😇。和杏菜结婚的话，女友😋，妻子😆，同学🤤三个愿望一次满足，实在是人生的至福🤗。啊，好想给杏菜戴上结婚戒指🤩，一边听着杏菜说【这些钱我也要还吗？】🧐，一边看着她把戒指当作一生宝物，一脸幸福的表情🤤。
![IMG_20240813_104022](https://github.com/user-attachments/assets/b613c910-aa72-4d80-87e8-d5ad50bb9aec)

**Comment by Volim07** [comment]
老八可爱捏

**Comment by ZZZ-MHY** [comment]
看这番我只想说有一个爱你的人不容易啊

**Comment by csjdz** [comment]
精神状态良好（确信）
